trl/trainer/utils.py

```diff
@@ -32,7 +32,9 @@
 from accelerate.state import AcceleratorState
 from huggingface_hub import ModelCard, ModelCardData
 from rich.console import Console
+from rich.panel import Panel
 from rich.table import Table
+from rich.text import Text
```

Comment:
It's a bit out of the scope, but I'd like to make rich an optional dep of trl. Can you use this instead:

```python
from ..import_utils import is_rich_available

if is_rich_available():
    from rich.console import Console
    from rich.panel import Panel
    from rich.table import Table
    from rich.text import Text

...

def print_prompt_completions_sample(...):
    if not is_rich_available():
        raise ImportError("...")
```

Reply:
I'll do you one better and add a fallback using regular print `print` :-)

Reply:
Ah, it seems like the pre-commit command has issues with the `print` statements. Can we replace them with something else?

Reply:
Nevermind, I removed it in the end